gateway-ha/src/main/java/io/trino/gateway/proxyserver/ProxyRequestHandler.java

```diff
@@ -179,7 +179,6 @@ private void performRequest(
         cookieBuilder.addAll(getOAuth2GatewayCookie(remoteUri, servletRequest));
 
         Request request = requestBuilder
-                .setPreserveAuthorizationOnRedirect(true)
```

Comment:
This capability was removed from airlift by https://github.com/airlift/airlift/pull/1316. It should not have any functional impact since redirects have always been returned to the client